feature/xhr/aggregate/index.js

```diff
@@ -75,6 +75,8 @@ function storeXhr(params, metrics, startTime, endTime, type) {
     return
   }
 
+  var useMetrics = startTime === undefined || startTime === null || startTime < 0 || !endTime || endTime < startTime
```

Comment:
I think this is redundant because having duration in metrics implies that we got the end time. The places where we set metrics.duration are the same places where we collect end time.

Reply:
For whatever reason, This was not true in the case that the loader and agg were mismatched. The metric data existed, the endTime passed into the method did not

Reply:
Hence why it stated working on Nike after this branch was pushed up to s3